### PR TITLE
Adding in install help for running your own copy

### PR DESCRIPTION
**Pull request by jonathanKingston**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,20 @@
...
+
+This dashboard provides a metrics dashboard to see the status of the [Observatory](https://observatory.mozilla.org/) for each link.
+
+
+## Install
+
+```
+pip install -r requirements.txt
+```
+
```
Comment (mozfreddyb) [inline README.md:11]: Should mention that you'd want to modify the sites.json in httpobsdashboard/conf.
Suggestion

```md
## Install and Setup
* Clone this repo
* `pip install -r requirements.txt`
* Add the websites you want to check to the JSON files in httpobsdashboard/conf/
```

Comment (jonathanKingston) [reply README.md:11]: I added a config heading in my amended commit, thanks 👍

File: README.md [hunk 2]
```diff
@@ -1 +1,26 @@
...
+## Running
+
+Running `make devserver` will make a live dev environment that can be loaded in the browser and refreshes every time you run `make generate` which fetches the reports from the Observatory.
```
Comment (april) [inline README.md]: It will also refresh anytime you change the templates, fyi.

Comment (jonathanKingston) [reply README.md]: Amended text in latest push and also fixed `make help` command listing.

**Pull request merged**